.github/workflows/pypi-release.yml

```diff
@@ -1,5 +1,9 @@
 name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
-on: [release]
+on:
+  release:
+    # only publish to pypi when gh release is published
+    # (there are many gh release triggers).
```

Comment:
I'm not sure what you mean by this? If there are many, do they do things differently? Why is it important to note that there are many?

Reply:
Thanks - I added this as a reference to this moment where we saw the job triggered from many types of release triggers. I'll remove the mention here as I think the meaning might be lost in time and become confusing.